src/components/Events/EventCard.tsx

```diff
@@ -6,32 +6,39 @@ import Image from 'next/image';
 import { StarRating } from '../StarRating';
 import { HeartButton } from '../HeartButton';
 import { Event } from '../../types/Event';
-import { Routes } from '../../utils/constants';
-import { formatDateTime } from '../../utils/dateUtils';
+import { Routes, levelLabels, sportsLabels } from '../../utils/constants';
 import { getImagePath } from '../../utils/functions';
+import { slugifyCategory } from '../../utils/slugifyUtils';
 
 const rating = 3.5;
 
 type EventCardProps = {
   event: Event;
+  index: number;
+  formattedDate: string;
 };
 
 export const EventCard: React.FC<EventCardProps> = ({
-  event: { id, name, sport, date, location, price, description, level },
+  event: { id, name, sport, location, price, description, level },
+  index,
+  formattedDate,
 }) => {
-  const formattedDate = formatDateTime(date);
+  const sportLabel = sportsLabels[sport] || 'Nezařazeno';
```

Comment:
Could it happen that it has no label? What about making a function `getSportLabel` that would simply return either sportLabel or the `Nezarazeno`? I have already seen the same logic on like 3 places.

Add it to utils, not constants :)

Reply:
Btw this might help for some additional type safety

```
const sportsArray = [
  'basketball',
  'running',
  'boxing',
  'cycling',
  'fitness',
  'football',
  'golf',
  'swimming',
  'skateboarding',
  'tableTennis',
  'tennis',
  'volleyball',
  'yoga',
] as const;

export const sportsSet = new Set(sportsArray);

type SportsType = (typeof sportsArray)[number];

export const sportsLabels: Record<SportsType, string> = {
  basketball: 'Basketbal',
  running: 'Běh',
  boxing: 'Box',
  cycling: 'Cyklistika',
  fitness: 'Fitness',
  football: 'Fotbal',
  golf: 'Golf',
  swimming: 'Plavání',
  skateboarding: 'Skateboarding',
  tableTennis: 'Stolní tenis',
  tennis: 'Tenis',
  volleyball: 'Volejbal',
  yoga: 'Yoga',
};
```